ios/FluentUI/AvatarGroup/AvatarGroup.swift

```diff
@@ -158,23 +158,6 @@ public struct AvatarGroup: View {
         self.tokens = state.tokens
     }
 
-    /// Renders the avatar with an optional cutout for the Stack group style.
-    @ViewBuilder
-    private func avatarCutout(_ avatar: Avatar,
```

Comment:
This is just dead code being removed, right? Doesn't look like it was being used anywhere.

Reply:
Right, looks like when we updated the animations of the group, we stopped using this.